source/virtualservice.go

```diff
@@ -317,6 +317,15 @@ func (sc *virtualServiceSource) setResourceLabel(virtualservice networkingv1alph
 	}
 }
 
+func appendIfMissing(targets []string, target string) []string {
```

Comment:
Do you mind adding some documentation around the purpose of this function?

Reply:
I just commited a change that improves the name of the function and adds a comment describing what it does.